raiden/raiden_service.py

```diff
@@ -478,13 +477,40 @@ def remove(_):
     def __repr__(self):
         return f'<{self.__class__.__name__} node:{pex(self.address)}>'
 
-    def start_neighbours_healthcheck(self, chain_state: ChainState):
+    def _initialize_transport(self, chain_state: ChainState):
+        """ Initialize the transport and related facilities.
+
+        Note:
+            The transport must not be started before the node has caught up
+            with the blockchain through `AlarmTask.first_run()`. This
+            synchronization includes the on-chain channel state and is
+            necessary to reject new messages for closed channels.
+        """
+        assert self.alarm.is_primed(), 'AlarmTask not primed. node:{self!r}'
+        assert not self.ready_to_process_events
+
+        self._initialize_messages_queues(chain_state)
```

Comment:
Before, whitelist was called before the message queue initialization. Does it matter that you changed it?